src/frontends/tensorflow/tests/requirements.txt

```diff
@@ -1,3 +1,5 @@
 -c ../../../bindings/python/constraints.txt
 numpy
 tensorflow
+# limit h5py version for Linux arm64, 3.11 version failed
+h5py~=3.10.0; sys_platform == 'linux' and platform_machine == 'aarch64'
```

Comment:
I am fine with this change.
What version of TF finally is installed? Do we preserve the expected version 2.15 or 2.16?

Reply:
No we don't restrict tf version:
Collecting tensorflow (from -r /__w/openvino/openvino/openvino/src/frontends/tensorflow/tests/requirements.txt (line 3))
  Using cached tensorflow-2.16.1-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl.metadata (4.1 kB)